v22.1/backup.md

```diff
@@ -140,7 +140,16 @@ This improves performance by decreasing the likelihood that the `BACKUP` will be
 
 {% include_cached new-in.html version="v22.1" %} A backup job will [pause](pause-job.html) instead of entering a `failed` state if it continues to encounter transient errors once it has retried a maximum number of times. Once the backup has paused, you can either [resume](resume-job.html) or [cancel](cancel-job.html) it.
 
-{% include {{ page.version.version }}/backups/file-size-setting.md %}
+### Backup performance configuration
+
+Cluster settings provide a means to tune a CockroachDB cluster. The following cluster settings are helpful for configuring backup files and performance:
+
+Setting    | Description
+-----------+-----------------
+`bulkio.backup.file_size` | Set a target for the amount of backup data written to each backup file.<br><br>**Default:** `128 MiB`
```

Comment:
Should we warn users against setting this value too low? If they set it too low then it will result in many small SSTs being created as part of the backup, which in turn could result in degraded restore performance because of the need to keep track of so many small files.

Reply:
Maybe we should also highlight that this is the maximum target size that we strive to hit, but it is possible that we create files of a smaller size than this for various reasons.

Reply:
Yes, definitely worthwhile. Thanks for providing the information here for that!

Reply:
@adityamaru I'd appreciate another quick review! I also changed up the formatting (sorry if that distracts!)